pages/common/firefox.md

```diff
@@ -27,6 +27,10 @@
 
 `firefox --headless --screenshot {{path/to/output_file.png}} {{https://example.com/}}`
 
-- Use a specific profile directory to allow multiple separate instances of Firefox to run at once:
+- Create a new profile in the default directory without starting Firefox:
 
-`firefox --new-instance --profile {{path/to/directory}} {{https://example.com/}}`
+`firefox --CreateProfile {{profile_name}}`
```

Comment:
Hold on. This wasn't quite what I was trying to suggest. What about just removing the `--new-instance`, and leaving the `--profile`?

Also, I'm not seeing any documentation for `--CreateProfile` - and the casing is different to the other arguments int he rest of the page. Could you clarify where this is from please?

Reply:
Here's the [specific documentation](https://developer.mozilla.org/en-US/docs/Mozilla/Command_Line_Options#-CreateProfile_profile_name) for --CreateProfile. I'm not sure why it's different casing, but there are a few others like [-setDefaultBrowser](https://developer.mozilla.org/en-US/docs/Mozilla/Command_Line_Options#-setDefaultBrowser) may be an old flag from Thunderbird as they share `--CreateProfile` but that doesn't explain the others...

Reply:
> Hold on. This wasn't quite what I was trying to suggest. What about just removing the `--new-instance`, and leaving the `--profile`?

My bad, I'll fix that now (double checked and it does still function correctly with just `--profile`)

Reply:
Thanks :-)